docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -21,22 +21,22 @@ This tutorial requires the following:
 
 and
 
-* The EOSIO platform software, Click on this link for instructions on [installing EOSIO binaries.](../02_getting-started/02_development-environment/03_before-you-begin.md "Getting started - install binaries") 
-* Familiarity with the EOSIO tools `cleos` and `keosd`.
-* The EOSIO Contract Development Toolkit or eosio.cdt. Click on this link to [install the CDT.](../02_getting-started/02_development-environment/04_install-the-CDT.md "EOSIO.CDT Installation Instructions") 
+* The Antelope framework software, Click on this link for instructions on [installing Antelope binaries.](../02_getting-started/02_development-environment/03_before-you-begin.md "Getting started - install binaries") 
+* Familiarity with the Antelope tools `cleos` and `keosd`.
+* The Antelope Contract Development Toolkit or eosio.cdt. Click on this link to [install the CDT.](../02_getting-started/02_development-environment/04_install-the-CDT.md "CDT Installation Instructions") 
 * Familiarity with the `eosio.cdt` which is used to compile smart contracts. 
 
-Click on this link for an [overview of the EOSIO platform.](../01_overview/index.md)
+Click on this link for an [overview of the Antelope framework.](../01_overview/index.md)
 
-Click on this link to [get started with EOSIO.](../02_getting-started/index.md "Getting Started with EOSIO")
+Click on this link to [get started with Antelope.](../02_getting-started/index.md "Getting Started with Antelope")
 
 ### Tutorial Workflow
 
 ```sequence
-Understand\n Game->Understand\n EOSIO : What?
-Understand\n EOSIO->Create\nAccounts  : Accounts
-Understand\n EOSIO->Create\nAccounts  : Actions
-Understand\n EOSIO->Create\nAccounts  : Resources
+Understand\n Game->Understand\n Antelope : What?
```

Comment:
We need to double-check this sequence diagram renders correctly. Created issue #16